applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -244,15 +231,13 @@ void UPwSmallStrainElement<TDim,TNumNodes>::
         }
     }
 
-    KRATOS_CATCH("");
+    KRATOS_CATCH("")
 }
 
-//----------------------------------------------------------------------------------------------------
 template< unsigned int TDim, unsigned int TNumNodes >
-void UPwSmallStrainElement<TDim,TNumNodes>::
-    InitializeNonLinearIteration(const ProcessInfo& rCurrentProcessInfo)
+void UPwSmallStrainElement<TDim,TNumNodes>::InitializeNonLinearIteration(const ProcessInfo& rCurrentProcessInfo)
```

Comment:
Sorry, I keep nitpicking:
```suggestion
void UPwSmallStrainElement<TDim, TNumNodes>::InitializeNonLinearIteration(const ProcessInfo& rCurrentProcessInfo)
```

Reply:
done